game_eggs/minecraft/java/mohist/egg-mohist.json

```diff
@@ -4,7 +4,7 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2021-06-14T21:28:24+03:00",
+    "exported_at": "2021-12-09T13:31:01-05:00",
```

Comment:
nothing was changed here

Reply:
yeap, sorry. removed the file in the commit below